x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors/utils.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type SuperTest from 'supertest';
+import { findIndex } from 'lodash';
+
+import { ObjectRemover } from '../../../lib/object_remover';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { getTestActionData } from '../../../lib/get_test_data';
+
+export const createConnectorAndObjectRemover = async ({
+  getPageObject,
+  getService,
+}: {
+  getPageObject: FtrProviderContext['getPageObject'];
+  getService: FtrProviderContext['getService'];
+}) => {
+  const supertest = getService('supertest');
+  const objectRemover = new ObjectRemover(supertest);
+
+  const testData = getTestActionData();
+  const createdAction = await createSlackConnector({
+    name: testData.name,
+    getPageObject,
+    getService,
+  });
+  objectRemover.add(createdAction.id, 'action', 'actions');
+
+  return objectRemover;
+};
+
+export const createSlackConnector = async ({
+  name,
+  getPageObject,
+  getService,
+}: {
+  name: string;
+  getPageObject: FtrProviderContext['getPageObject'];
+  getService: FtrProviderContext['getService'];
+}) => {
+  const connector = await createConnector({
+    name,
+    config: {},
+    secrets: { webhookUrl: 'https://test.com' },
+    connectorTypeId: '.slack',
+    getPageObject,
+    getService,
+  });
+
+  return connector;
+};
+
+export const getConnector = async (
```

Comment:
Maybe we should use get the connector by ID: https://www.elastic.co/guide/en/kibana/current/get-connector-api.html

Reply:
The reason to use this function is when we don't have the connector id. A few of the tests create the connector via the UI instead of an API call.

Reply:
I'll update the name to `getConnectorByName`

Reply:
Oh, I see. Thanks!